AnkiDroid/src/main/java/com/ichi2/compat/CompatHelper.kt

```diff
@@ -37,6 +37,7 @@ class CompatHelper private constructor() {
         sdkVersion >= Build.VERSION_CODES.Q -> CompatV29()
         sdkVersion >= Build.VERSION_CODES.O -> CompatV26()
         sdkVersion >= Build.VERSION_CODES.M -> CompatV23()
+        sdkVersion >= Build.VERSION_CODES.R -> CompatV30()
```

Comment:
```suggestion
        sdkVersion >= Build.VERSION_CODES.R -> CompatV30()
        sdkVersion >= Build.VERSION_CODES.Q -> CompatV29()
        sdkVersion >= Build.VERSION_CODES.O -> CompatV26()
        sdkVersion >= Build.VERSION_CODES.M -> CompatV23()
```

These must be in decreasing order or the fall through doesn't work correctly

Reply:
Done.